docs/kyma/docs/036-inst-xip.io-domain.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Install Kyma on a GKE cluster with wildcard DNS
+type: Installation
+---
+
+If you want to try Kyma in a cluster environment without assigning the cluster to a domain you own, you can use [`xip.io`](www.xip.io) which provides a wildcard DNS for any IP address. Such scenario requires using a self-signed TLS certificate.
+
+This solution is not suitable for a production environment, but makes for a great playground which allows to get to know the product better.
+
+## Prerequisites
+
+The prerequisites match these listed in the **Install Kyma on a GKE cluster** document. However, you don't need to prepare a domain for your cluster as it is replaced by a wildcard DNS provided by [`xip.io`](www.xip.io).
```

Comment:
Wouldn't it be more convenient for the user to have all the prerequisites listed here instead of looking for a separate document?

Reply:
Let's not duplicate the content that's in another document - it's gonna make it harder to maintain. When linking is available it's going to be much more convenient to jump between these two docs. 
The prereqs might change and if that happens then you'd have to adjust them in two places instead of one. It can cause inconsistency in our docs and confuse the user much more than having to refer to another doc for details ;)